RecoPixelVertexing/PixelLowPtUtilities/test/ClusterShapeExtractor.cc

```diff
@@ -46,405 +46,370 @@ using namespace std;
+
+  void processPixelRecHits(const SiPixelRecHitCollection::DataContainer* recHits,
+                           const SiPixelClusterShapeCache& clusterShapeCache);
+  void processStripRecHits(const SiStripRecHit2DCollection::DataContainer* recHits);
+  void processMatchedRecHits(const SiStripMatchedRecHit2DCollection::DataContainer* recHits);
+
+  void analyzeSimHits(const edm::Event& ev, const edm::EventSetup& es);
+  void analyzeRecTracks(const edm::Event& ev, const edm::EventSetup& es);
+
+  TFile* file;
+
+  string trackProducer;
+  bool hasSimHits;
+  bool hasRecTracks;
+
+  edm::EDGetTokenT<edmNew::DetSetVector<SiPixelRecHit>> pixelRecHits_token;
+  edm::EDGetTokenT<SiPixelClusterShapeCache> clusterShapeCache_token;
+
+  const TrackerGeometry* theTracker;
+  std::unique_ptr<TrackerHitAssociator> theHitAssociator;
+  TrackerHitAssociator::Config trackerHitAssociatorConfig_;
+  const ClusterShapeHitFilter* theClusterShape;
+
+  vector<TH2F*> hspc;  // simulated pixel cluster
+  vector<TH1F*> hssc;  // simulated strip cluster
+
+  vector<TH2F*> hrpc;  // reconstructed pixel cluster
+  vector<TH1F*> hrsc;  // reconstructed strip cluster
 };
 
 /*****************************************************************************/
-void ClusterShapeExtractor::beginRun(const edm::Run & run, const edm::EventSetup& es)
-{
+void ClusterShapeExtractor::beginRun(const edm::Run& run, const edm::EventSetup& es) {
   // Get tracker geometry
-  edm::ESHandle<TrackerGeometry>          tracker;
+  edm::ESHandle<TrackerGeometry> tracker;
   es.get<TrackerDigiGeometryRecord>().get(tracker);
-  theTracker =                            tracker.product();
+  theTracker = tracker.product();
 
-  // 
+  //
   //  theClusterShape = new ClusterShapeHitFilter(es);
   edm::ESHandle<ClusterShapeHitFilter> shape;
-  es.get<CkfComponentsRecord>().get("ClusterShapeHitFilter",shape);
+  es.get<CkfComponentsRecord>().get("ClusterShapeHitFilter", shape);
   theClusterShape = shape.product();
 
   // Declare histograms
   char histName[256];
 
   // pixel
-  for(int subdet = 0; subdet <= 1; subdet++)
-  {
-    for(int ex = 0; ex <= exMax; ex++)
-    for(int ey = 0; ey <= eyMax; ey++)
-    {
-      sprintf(histName,"hspc_%d_%d_%d",subdet, ex,ey);
-      hspc.push_back(new TH2F(histName,histName, 
-                         10 * 2 * (exMax+2), -(exMax+2),(exMax+2),
-                         10 * 2 * (eyMax+2), -(eyMax+2),(eyMax+2)));
-
-      sprintf(histName,"hrpc_%d_%d_%d",subdet, ex,ey);
-      hrpc.push_back(new TH2F(histName,histName, 
-                         10 * 2 * (exMax+2), -(exMax+2),(exMax+2),
-                         10 * 2 * (eyMax+2), -(eyMax+2),(eyMax+2)));
-    }
+  for (int subdet = 0; subdet <= 1; subdet++) {
+    for (int ex = 0; ex <= exMax; ex++)
+      for (int ey = 0; ey <= eyMax; ey++) {
+        sprintf(histName, "hspc_%d_%d_%d", subdet, ex, ey);
+        hspc.push_back(new TH2F(histName,
```

Comment:
Also here: when lines do not exceed the maximum length, why not to trust the line splitting proposed by the author of the code, which is in most cases aesthetically better than the automatically reformatted one?